k8s/configmap.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flink-config
+  labels:
+    app: flink
+data:
+  flink-conf.yaml: |+
+    parallelism.default: 2
+    jobmanager.memory.process.size: 4048m
+    taskmanager.memory.process.size: 4048m
```

Comment:
The memory seems a bit odd. How did you arrive at `4048`?

Reply:
This is just an initial value that would be changed. No logic is there behind this.

Reply:
By why not `4096`? 4048 seems a bit odd.

Reply:
Changed.